content/projects/gsoc/2021/project-ideas/tekton-client-plugin.adoc

```diff
@@ -1,47 +1,48 @@
 ---
 layout: gsocprojectidea
-title: "Tekton client plugin for Jenkins"
-goal: "Trigger Tekton Pipelines from Jenkins Jobs."
+title: "Tekton Pipelines DSL for Jenkins with the Tekton Client Plugin"
+goal: "Manage Tekton resources via Pipeline Steps / Jenkinsfile"
 category: Plugins
 year: 2021
 status: draft
 sig: docs
 skills:
 - Java
 - Kubernetes
-- Tekton
+- Tekton Pipelines
 mentors:
 - "waveywaves"
 - "garethjevans"
 links:
   draft: https://docs.google.com/document/d/1_mS0gLgM7oz7PanE0X_SkJ8iUo-BPVCixXMSDO7-VMk/edit#
+  design doc: https://docs.google.com/document/d/10n3vgYrjS_M1571OUOYLw4tK2QmxY7kp_wyUT3ke4gY/edit#
```

Comment:
I don't see this being made visible anywhere when the page is rendered.  Since it is the same link that is also included in the `Links` section, I don't object to this, just don't see it as visible when I view the page from my dev env.
```suggestion
  design doc: https://docs.google.com/document/d/10n3vgYrjS_M1571OUOYLw4tK2QmxY7kp_wyUT3ke4gY/edit#
```